packages/api/cms-api/src/index.ts

```diff
@@ -218,6 +218,7 @@ export { ImgproxyModule } from "./imgproxy/imgproxy.module";
 export { ImgproxyConfig, ImgproxyService } from "./imgproxy/imgproxy.service";
 export { KubernetesJobStatus } from "./kubernetes/job-status.enum";
 export { KubernetesModule } from "./kubernetes/kubernetes.module";
+export { KubernetesService } from "./kubernetes/kubernetes.service";
```

Comment:
If we decide to not do this, we should still consider to export the Kubernetes service so applications can build something like this on their own.